tests/phpunit/tests/media.php

```diff
@@ -3617,6 +3618,16 @@ private function reset_omit_loading_attr_filter() {
 		// Clean up the above filter.
 		remove_filter( 'wp_omit_loading_attr_threshold', '__return_null', 100 );
 	}
+
+	/**
+	 * Test the wp_default_image_output_mapping function.
+	 *
+	 * @ticket 55443
+	 */
+	public function test_wp_default_image_output_mapping() {
+		$mapping = wp_default_image_output_mapping();
+		$this->assertSame( array( 'image/jpeg' => 'image/webp' ), $mapping );
+	}
```

Comment:
This test is a good starting point (pending adjustments because we probably need to pass a parameter to the function), but we should cover a few more things (partly related to my above comments).

I would suggest adding at least 2 more tests:
* One where you pass an array with some other image mapping into the function to ensure it's not overwritten by it.
* One where you actually generate a new version of a JPEG image, e.g.
    * Calling `wp_get_image_editor()` for some test data JPEG
    * Resizing it down to some arbitrary size
    * Ensuring it comes out as WebP (via the actual filter and the `wp_default_image_output_mapping` callback hooked into it)

Reply:
Added these 👍🏼